package.json

```diff
@@ -49,7 +49,7 @@
     "download": "babel-node scripts/download.js cache/data.json",
     "build-public": "browserify --standalone Schema -t babelify --outfile public/schema.js src/schema-proxy.js",
     "serve-public": "babel-node scripts/serve-public",
-    "deploy": "scripts/build-public && scripts/deploy-public",
+    "deploy": "yarn run build-public && scripts/deploy-public",
```

Comment:
@acao I think I merged it to fast.
Other scripts in `package.json` use `npm run` not `yarn run` so I think this one should do the same.